packages/ariakit-react-utils/package.json

```diff
@@ -33,6 +33,9 @@
     "react",
     "utils"
   ],
+  "dependencies": {
+    "ariakit-utils": "0.17.0-next.26"
+  },
```

Comment:
Nice 👍 
Could you do that in separate PR, please?

Reply:
Yup, sure thing!